config/production.json

```diff
@@ -0,0 +1,4 @@
+{
+  "mongoURI": "mongodb+srv://poke123:poke123@pokemonkeeper-hlpd2.mongodb.net/test?retryWrites=true&w=majority",
+  "jwtSecret": "pokesecret"
```

Comment:
I wouldn't add such values to a public Github repo 😛 It's an example project so it's fine, but it's generally a bad practice to save sensitive data in github repo (especially public one)